src/devices/Amg88xx/Amg88xx.cs

```diff
@@ -95,6 +95,29 @@ public Amg88xx(I2cDevice i2cDevice)
             }
         }
 
+        /// <summary>
+        /// Gets temperature for all pixels from the current thermal image as a two-dimensional array.
+        /// First index specifies the x-coordinate of the pixel and second index specifies y-coordinate of the pixel.
+        /// </summary>
+        /// <returns>Temperature as a two-dimensional array.</returns>
+        public Temperature[,] TemperatureMatrix
+        {
+            get
+            {
+                Span<byte> buffer = _imageData;
```

Comment:
This doesn't seem to be used?

Reply:
Absolutely right. Leftover of a refactoring.